tools/bcl-test-importer/BCLTestImporter/RegisterTypeGenerator.cs

```diff
@@ -38,7 +81,7 @@ public static class RegisterTypeGenerator {
 				return result;
 			}
 		}
-		
+
```

Comment:
^ lots of unrelated noise
please remove them if the PR needs any update before merging

Reply:
will clean those.

Reply:
Cleaned in the last push.